CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-# 1.1.1
+# 1.2.0
+
+- [upgrade finagle-postgres to 0.4.2](https://github.com/getquill/quill/pull/759)
+- [add collections support for row elements (SQL Arrays, Cassandra Collection)](https://github.com/getquill/quill/pull/758)
+- [add querySchema support to optional nested case class](https://github.com/getquill/quill/pull/767)
```

Comment:
Could you please rename it to:
`allow querySchema/schemaMeta to rename optional embedded case classes` - imho, it better described changes